src/bootctl/bootctl-install.c

```diff
@@ -335,7 +335,10 @@ static int create_subdirs(const char *root, const char * const *subdirs) {
         return 0;
 }
 
-static int update_efi_boot_binaries(const char *esp_path, const char *source_path) {
+static int update_efi_boot_binaries(
+                const char *esp_path,
+                const char *source_path,
+                const char *ignore_filename) {
         _cleanup_closedir_ DIR *d = NULL;
```

Comment:
Not a strict rule, but we usually insert an empty line between function arguments and variable declarations.